cl/scrapers/tasks.py

```diff
@@ -650,3 +660,198 @@ def update_docket_info_iquery(self, d_pk: int, court_id: str) -> None:
+    could not be decoded, or the subscription process failed for any other reason.
+    """
+    if settings.OPENAI_TRANSCRIPTION_KEY is None:
+        logger.error(
+            "OPENAI_TRANSCRIPTION_KEY environment variable is not set."
+        )
+        raise ScrapeFailed(
+            "OPENAI_TRANSCRIPTION_KEY environment variable is not set."
+        )
+
+    docket = Docket.objects.get(pk=pk)
+    docket_number = docket.docket_number
+
+    session = requests.Session()
+    session.headers.update(
+        {
+            "User-Agent": "Free Law Project",
+        }
+    )
+
+    base_url = "https://file.supremecourt.gov"
+    form_url = f"{base_url}/CaseNotification?caseNumber={docket_number}"
+    try:
+        logger.info("Fetching subscription page for case %s", docket_number)
+        response = session.get(form_url, timeout=10)
+        response.raise_for_status()
+        scotus_html = BeautifulSoup(response.content, "html.parser")
+        form = scotus_html.find("form", id="CaseNotificationForm")
+        if not form:
+            raise ScrapeFailed("Could not find the main subscription form.")
+
+        # Collect all form inputs and include the anti-forgery token and CaseNumber
+        payload = {}
+        for input_tag in form.find_all("input"):
+            name = input_tag.get("name")
+            value = input_tag.get("value", "")
+            if name:
+                payload[name] = value
+
+        anti_forgery_token = payload.get("__RequestVerificationToken")
+        if not anti_forgery_token:
+            raise ScrapeFailed("Could not find __RequestVerificationToken.")
+
+        captcha_reset_url = f"{base_url}/Captcha/Reset"
+        captcha_payload = {"__RequestVerificationToken": anti_forgery_token}
+        reset_response = session.post(
+            captcha_reset_url,
+            data=captcha_payload,
+            headers={
+                "Referer": form_url,
+                "X-Requested-With": "XMLHttpRequest",
+            },
+            timeout=10,
+        )
+        reset_response.raise_for_status()
+        reset_data = reset_response.json()
+        captcha_id = reset_data.get("captchaId")
+        if not captcha_id:
+            raise ScrapeFailed(
+                f"Failed to get captchaId from /Captcha/Reset. Response: {reset_response.text}"
+            )
+
+        # Fetch the Audio
+        audio_url = f"{base_url}/Captcha/audio?captchaId={captcha_id}"
+        audio_response = session.get(
+            audio_url, headers={"Referer": form_url}, timeout=10
+        )
+        audio_response.raise_for_status()
+
+        # Solve the captcha
+        audio_file = BytesIO(audio_response.content)
+        transcription = call_llm_transcription(
+            ("captcha.wav", audio_file),
+            api_key=settings.OPENAI_TRANSCRIPTION_KEY,
+        )
+        solution = process_scotus_captcha_transcription(transcription)
+        logger.info("Solved CAPTCHA: %s", solution)
```

Comment:
I think we can remove this log. Or do you see any scenario where it could be helpful?